app/assets/javascripts/deck_selection.js.coffee

```diff
@@ -1,3 +1,6 @@
+#= require deck_model
```

Comment:
remove commented code

Reply:
oh, wait.  is this how coffeescript actually handles requires?  ignore these comments if so.

Reply:
It's not CoffeeScript, it's the Rails asset pipeline. It's used to decide which order to put the scripts in when it builds the assets, otherwise there's no particular order.

These directives fix a bug which meant none of the decks ever displayed when you use production deployment, because the code was loaded in the wrong order. In the development environment, it loads them alphabetically, but in a production environment the order is different, maybe based on a hash or something.